src/doc/book/ffi.md

```diff
@@ -521,14 +521,14 @@ against `libc` and `libm` by default.
 
 # The "nullable pointer optimization"
 
-Certain types are defined to not be `null`. This includes references (`&T`,
+Certain types are defined to not be `NULL`. This includes references (`&T`,
```

Comment:
Why is it back ticked here? It's definitely not an identifier that Rust uses, so it seems wrong.